.github/workflows/setupForWlsVm.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+################################################
+# This script is invoked by a human who:
+# - has done az login.
+# - can create repository secrets in the github repo from which this file was cloned.
+# - has the gh client >= 2.0.0 installed.
+#
+# This script initializes the repo from which this file is was cloned
+# with the necessary secrets to run the workflows.
+#
+# Script design taken from https://github.com/microsoft/NubesGen.
+#
+################################################
+
+################################################
+# Set environment variables - the main variables you might want to configure.
+#
+AKS_REPO_USER_NAME=oracle
+# Three letters to disambiguate names.
+DISAMBIG_PREFIX=
+# URI (hostname:port) for Elastic server, leave blank if you don't want to integrate ELK.
+ELK_URI=
+# Account name for Elastic server, leave blank if you don't want to integrate ELK.
+ELK_USER_NAME=
+# Account password for Elastic server, leave blank if you don't want to integrate ELK.
+ELK_PSW= 
+# The location of the resource group. For example `eastus`. Leave blank to use your default location.
+LOCATION=
+# Oracle single sign-on userid.
+OTN_USERID=
+# Password for preceding Oracle single sign-on userid.
+OTN_PASSWORD=
+# User Email of GitHub acount to access GitHub repository.
+USER_EMAIL=
+# User name for preceding GitHub account.
+USER_NAME=
+# Personal token for preceding GitHub account.
+GIT_TOKEN=
+WLS_PSW=Secret123456
```

Comment:
Is this a secret name or a passwd?

Reply:
Not really, but I'll fix it to prompt the user for it right now.

Reply:
Good catch. I am fixing this now.